graphql/schemabuilder/batch.go

```diff
@@ -274,6 +290,10 @@ func (funcCtx *batchFuncContext) consumeReturnValue(m *method, sb *schemaBuilder
 	return retType, out, nil
 }
 
+func isBatchIndexType(t reflect.Type) bool {
+	return t.PkgPath() == "github.com/samsarahq/thunder/batch" && t.String() == "batch.Index"
```

Comment:
does this need to check for vendoring?

Reply:
yuuuuuuup

Reply:
```
v := reflect.ValueOf(graphql.Executor{})
fmt.Println(v.Type().PkgPath())
```
`.../vendor/github.com/samsarahq/thunder/graphql`

Reply:
Updated to a `strings.Contains` instead.

Reply:
can't it do a direct type equality check?

```
var batchIndexPointer *batch.Index
batchIndexTyp := reflect.New(reflect.TypeOf(batchIndexPointer)).Elem()
```

Reply:
ah, yes, let's do that

Reply:
done